### PR TITLE
Update troubleshoot guide and pricing info

### PR DESCRIPTION
**Pull request by alch-msft**
Update troubleshoot guide and pricing info

### CODE REVIEW

File: academic-services/knowledge-exploration-service/resources-pricing.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ description: Provides Azure cost estimator links to model the cost of using Micr
...
 
 API Purpose | Description | Host Machine Sku | Cost estimate
 --- | --- | --- | ---
-Sub-graph access | For enterprise or domain specific solutions that only require a smaller sub-graph of the full Microsoft Academic Graph (MAG). | Standard_DS4_v3  | [Pricing calculator](https://azure.com/e/c04290edd8bf4db08ba23db8392430c4)
-Full graph access | Provides moderately fast, responsive API method response times over the entirety of the Microsoft Academic Graph (MAG). |  Standard_DS14_v2 | [Pricing calculator](https://azure.com/e/9bfe795705424b26a118639a198adfbd)
-Fast, full graph access | For production scenarios that require very fast, responsive API method response times over the entirety of the Microsoft Academic Graph (MAG).  | Standard_E32s_v3 |[Pricing calculator](https://azure.com/e/2c1ee1b19db84b7dbb6d3eb7b5492d6c)
+Sub-graph access | For enterprise or domain specific solutions that only require a smaller sub-graph of the full Microsoft Academic Graph (MAG). | Standard_DS4_v3  | [Pricing calculator](https://azure.com/e/4a67bc36bc864ed5ac73d4c4724f0886)
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-pricing.md:13]: >Standard_DS4_v3 [](start = 147, length = 15)

Shouldn't that be Standard_D4_v3? I thought the DS series were the ones with smaller temp storage

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 2]
```diff
@@ -19,13 +19,25 @@ You can find this information by logging into the Azure Management Portal, searc
 
 ## Azure Subscription quota limit reached
 
-If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you'll need to contact Azure support to increase your account quota, try a different region where you have more Azure quota, or change the MAKES host machine sku/worker machine sku settings. 
+If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you can to contact Azure support to increase your account quota, try a different region where you have more Azure quota, or change the MAKES host machine SKU/worker machine SKU parameter. 
 
-For most Microsoft Academic sub-graph index builds, you can scale down to **Standard_D8_v3** sku to build and host the indexes. MAKES indexing resources will scale using Azure Batch's "Dedicated Cores". Make sure you're requesting for the correct quota increase for the correct resource and region.
+When running **CreateHostResources** command:
+
+    Requires at least your subscription to have at least 4 avaliable vCPU cores and ability to create **Standard_DS4_v2** Virtual Machines.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:26]: >Requires at least your subscription [](start = 4, length = 35)

"Requires your subscription"

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 3]
```diff
@@ -19,13 +19,25 @@ You can find this information by logging into the Azure Management Portal, searc
...
+If you encounter errors such as "operation could not be completed as it results in exceeding approved XXXXXX quota", you can to contact Azure support to increase your account quota, try a different region where you have more Azure quota, or change the MAKES host machine SKU/worker machine SKU parameter. 
 
-For most Microsoft Academic sub-graph index builds, you can scale down to **Standard_D8_v3** sku to build and host the indexes. MAKES indexing resources will scale using Azure Batch's "Dedicated Cores". Make sure you're requesting for the correct quota increase for the correct resource and region.
+When running **CreateHostResources** command:
+
+    Requires at least your subscription to have at least 4 avaliable vCPU cores and ability to create **Standard_DS4_v2** Virtual Machines. 
+
+When running **DeployHost** command:
+
+    By default, uses Virtual Machine SKU **Standard_D14_v2**. (Running the command by default requires your subscription to have access to at least 16 vCPU and ability to Virtual Machine with **Standard_D14_v2** SKU in west us region.) For most Microsoft Academic sub-graph index, you can scale down to **Standard_D8_v3** sku to host the indexes.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:30]: >2 [](start = 57, length = 1)

v3

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 4]
```diff
@@ -35,15 +47,17 @@ A release may not be completed when you see a new folder in your storage account
 
 ## MAKES hosting resource/deployment region mismatch
 
-MAKES can only be deployed to the same region as the MAKES hosting image. If you see an error message containing "...The Image '/subscriptions/XXXXXXX-XXXX-XXX-XXXX-XXXXXXX/resourceGroups/xxxxmakeshostres/providers/Microsoft.Compute/images/xxxxmakeshostres' cannot be found in 'xxxx' region..." Make sure to create MAKES deployments and hosting images in the same region.
+When running **DeployHost** command, If you encounter an error messages containing "...The Image '/subscriptions/XXXXXXX-XXXX-XXX-XXXX-XXXXXXX/resourceGroups/xxxxmakeshostres/providers/Microsoft.Compute/images/xxxxmakeshostres' cannot be found in 'xxxx' region...", make sure to create MAKES deployments and hosting images in the same region.
+
+MAKES host can only be deployed the same region as the MAKES hosting image. Use the same --Region parameter as you did when running **CreateHostResources** command.
 
-## Wrong MAKES path parameters
+## Melformed path parameters
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:54]: Malformed

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md (hunk 3 above)
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:30]: >and ability to Virtual Machine with **Standard_D14_v2** SKU in west us region. [](start = 156, length = 78)

"and the ability to create Virtual Machine instances using **Standard_D14_v3** SKU in the West US region"

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:30]: >For most Microsoft Academic sub-graph index, you can scale down to **Standard_D8_v3** sku to host the indexes. [](start = 236, length = 110)

"For most sub-graph indexes, you can scale down to use **Standard_D8_v3** SKU instances to host the indexes."

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 5]
```diff
@@ -19,13 +19,25 @@ You can find this information by logging into the Azure Management Portal, searc
...
+
+    Requires at least your subscription to have at least 4 avaliable vCPU cores and ability to create **Standard_DS4_v2** Virtual Machines. 
+
+When running **DeployHost** command:
+
+    By default, uses Virtual Machine SKU **Standard_D14_v2**. (Running the command by default requires your subscription to have access to at least 16 vCPU and ability to Virtual Machine with **Standard_D14_v2** SKU in west us region.) For most Microsoft Academic sub-graph index, you can scale down to **Standard_D8_v3** sku to host the indexes. 
+
+When running **BuildIndex** command:
+
+    You can use the default Virtual Machine SKU **Standard_D8_v3** for most Microsoft Academic sub-graph index builds. You can change the configuration to **Standard_D14_v2** or larger machines for faster builds, if you already have access to the number of vCPU and Virtual Machine SKU.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:34]: >2 [](start = 171, length = 1)

v3

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:34]: >machines [](start = 185, length = 8)

SKU's

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:34]: >, if you already have access to the number of vCPU and Virtual Machine SKU [](start = 211, length = 74)

I'm not sure what this means

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 6]
```diff
@@ -35,15 +47,17 @@ A release may not be completed when you see a new folder in your storage account
 
 ## MAKES hosting resource/deployment region mismatch
 
-MAKES can only be deployed to the same region as the MAKES hosting image. If you see an error message containing "...The Image '/subscriptions/XXXXXXX-XXXX-XXX-XXXX-XXXXXXX/resourceGroups/xxxxmakeshostres/providers/Microsoft.Compute/images/xxxxmakeshostres' cannot be found in 'xxxx' region..." Make sure to create MAKES deployments and hosting images in the same region.
+When running **DeployHost** command, If you encounter an error messages containing "...The Image '/subscriptions/XXXXXXX-XXXX-XXX-XXXX-XXXXXXX/resourceGroups/xxxxmakeshostres/providers/Microsoft.Compute/images/xxxxmakeshostres' cannot be found in 'xxxx' region...", make sure to create MAKES deployments and hosting images in the same region.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:50]: >make sure to create MAKES deployments and hosting images in the same region [](start = 266, length = 75)

"make sure that both your MAKES host image resources and deployments are in the same region"

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md (hunk 4 above)
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:54]: >Melformed [](start = 3, length = 9)

Malformed
(hah)

**Comment by DarrinEide** [comment]
> ms.date: 2020-04-15

Update timestamp

---
Refers to: academic-services/knowledge-exploration-service/get-started-create-api-instances.md:5 in 78085b6. [](commit_id = 78085b640745c9fcb71485e8d14bbea2e3cdd090, deletion_comment = False)

**Comment by DarrinEide** [comment]
> ms.date: 2020-04-15

Update timestamp

---
Refers to: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:5 in 78085b6. [](commit_id = 78085b640745c9fcb71485e8d14bbea2e3cdd090, deletion_comment = False)

File: academic-services/knowledge-exploration-service/resources-pricing.md [hunk 7]
```diff
@@ -4,15 +4,15 @@ description: Provides Azure cost estimator links to model the cost of using Micr
 ms.topic: reference
 ms.date: 2020-02-10
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-pricing.md:5]: >2020-02-10 [](start = 9, length = 10)

Update timestamp

File: academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ ms.topic: reference
 ms.date: 2020-04-15
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/resources-troubleshoot-guide.md:5]: >2020-04-15 [](start = 9, length = 10)

Update timestamp

**Pull request merged**